RecipeBook/ViewControllers/HomeScreenViewController/HomeScreenViewController.swift

```diff
@@ -91,18 +99,48 @@ extension HomeScreenViewController {
 
 }
 
-extension HomeScreenViewController: UICollectionViewDataSource {
+// MARK: Recommendation Collection View
+extension HomeScreenViewController {
 
-    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        return viewModel.numberOfItemsInSection
+    func setupRecommendationsCollectionView() {
+        recommendationsCollectionView.delegate = self
+        recommendationsCollectionView.dataSource = self
+        recommendationsCollectionView.backgroundColor = .clear
+        recommendationsCollectionView.showsHorizontalScrollIndicator = false
+        recommendationsCollectionView.collectionViewLayout = UICollectionViewFlowLayout()
+        if let layout = recommendationsCollectionView.collectionViewLayout as? UICollectionViewFlowLayout {
+            layout.scrollDirection = .horizontal
+            layout.minimumInteritemSpacing = 0
+            layout.minimumLineSpacing = 16
+            layout.sectionInset = UIEdgeInsets(top: 0, left: 16, bottom: 0, right: 16)
+        }
     }
+}
+
+extension HomeScreenViewController: UICollectionViewDataSource {
+
+    func collectionView(
+        _ collectionView: UICollectionView,
+        numberOfItemsInSection section: Int) -> Int {
+            if collectionView == self.categoriesCollectionView {
+
+                return viewModel.categoriesNumberOfItemsInSection
+            }
+            return viewModel.recommendationsNumberOfItemsInSection
```

Comment:
When we deal with multiple collection views, the best practice is to use an `if` statement for each one.